src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -118,6 +118,15 @@ public MtAmazonDynamoDbBySharedTable(String name,
         this.truncateOnDeleteTable = truncateOnDeleteTable;
     }
 
+    List<CreateTableRequest> getSharedTables() {
+        return tableMappingFactory.getCreateTableRequestFactory().getPhysicalTables();
+    }
+
+    @Override
+    protected MtAmazonDynamoDbContextProvider getMtContext() {
+        return super.getMtContext();
+    }
```

Comment:
Need access to context in MtStreams. Base class declares method as protected in different package, so had to override to make it visible.